ansible/roles/ansible-tvault-contego-extension/tasks/contego_service.yml

```diff
@@ -1,26 +1,25 @@
-- debug: msg="ansible distribution is:{{ansible_distribution}}"
-
+---
 - name: get configuration parameters from {{TVAULT_CONTEGO_CONF}}
   copy: src={{files_path}}/files/get_configs.py dest=/tmp/get_configs.py
 
 - name: run /tmp/get_configs.py
-  shell: "sudo -u {{TVAULT_CONTEGO_EXT_USER}} python /tmp/get_configs.py"
+  shell: "{{virtual_env}} && sudo -u {{TVAULT_CONTEGO_EXT_USER}} python3 /tmp/get_configs.py {{NOVA_CONF_FILE}}"
```

Comment:
don't we need to check if python2 or python3 needed here ?

Reply:
and don't we need to activate the virtual_env ? 
this comment applies to every other place where it is mentioned like this.

Reply:
changes done